pkg/sdn/plugin/controller.go

```diff
@@ -176,85 +178,84 @@ func deleteLocalSubnetRoute(device, localSubnetCIDR string) {
 	}
 }
 
-func (plugin *OsdnNode) SetupSDN() (bool, error) {
-	localSubnetCIDR, err := plugin.getLocalSubnet()
-	if err != nil {
-		return false, err
+// Detect whether we are upgrading from a pre-CNI openshift and clean up
+// interfaces and iptables rules that are no longer required
+func (plugin *OsdnNode) dockerCleanup(localSubnetCIDR string) error {
+	cleanup := false
+
+	interfaces := []string{"lbr0", "vovsbr", "vlinuxbr"}
+
+	exec := kexec.New()
+	for _, intf := range interfaces {
```

Comment:
You don't need to test all three; we only need to handle transitions from working configurations.